src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -147,7 +147,7 @@ def create_role_assignment(cmd, role, assignee=None, assignee_object_id=None, re
         raise CLIError('usage error: --assignee STRING | --assignee-object-id GUID')
 
     if assignee_principal_type and not assignee_object_id:
-        raise CLIError('usage error: --assignee-object-id GUID [--assignee-principal-type]')
+        raise CLIError('usage error: --assignee-object-id GUID --assignee-principal-type TYPE')
```

Comment:
As `--assignee-object-id` is initially designed to bypass Graph call, we encourage the user to use `--assignee-object-id` along with `--assignee-principal-type`. Specifying `--assignee-object-id` without `--assignee-principal-type` is only for backward compatibility.